gr-utils/python/modtool/templates/gr-newmod/cmake/Modules/GrPython.cmake

```diff
@@ -38,11 +38,11 @@ else(PYTHON_EXECUTABLE)
 
     #use the built-in find script
     set(Python_ADDITIONAL_VERSIONS 3.4 3.5 3.6)
-    find_package(PythonInterp 2)
+    find_package(PythonInterp 3)
 
     #and if that fails use the find program routine
     if(NOT PYTHONINTERP_FOUND)
-        find_program(PYTHON_EXECUTABLE NAMES python python2 python2.7)
+        find_program(PYTHON_EXECUTABLE NAMES python python3 python3.6)
```

Comment:
why only `python3.6` here, no `python3.4` etc?

Reply:
I have added the previous versions as well. :)